tarantool/connection.py

```diff
@@ -363,7 +363,8 @@ def __init__(self, host, port,
                  ssl_ca_file=DEFAULT_SSL_CA_FILE,
                  ssl_ciphers=DEFAULT_SSL_CIPHERS,
                  packer_factory=default_packer_factory,
-                 unpacker_factory=default_unpacker_factory):
+                 unpacker_factory=default_unpacker_factory,
+                 fetch_schema=True):
```

Comment:
Since `ConnectionPool` does not inherit `Connection` (I had reasons for that), you need to add new parameter explicitly.

Reply:
Implemented